isa/ldot-bdot/ldot-bdot.adoc

```diff
@@ -671,12 +671,14 @@ The invalid operation flag must be raised if at least one of the following condi
 
 - Any of the operands is a signaling NaN
 - At least one of the following conditions:
-**** there are at least two products that are infinites with opposite sign
-**** there is at least one product between a zero and an infinity
+**** there are at least two products that are infinites with opposite sign.
+**** there is at least one product between a zero and an infinity.
 
-Note:: a product is said to be infinite when it is the product between an infinity and a non-zero finite number.
+// comment to force next notes to be left aligned
```

Comment:
Without this comment to break the list alignment (suggested by https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#lists), the notes are deported right
![image](https://github.com/user-attachments/assets/7b8d2188-bfff-4c99-b2f4-9cefda33e7ab)

Reply:
Good ol' asciidoc.